python/src/worker/helpers/worker_exception.py

```diff
@@ -0,0 +1,4 @@
+class WorkerException(Exception):
```

Comment:
Custom exceptions should be defined package-wide to make sure they are all in the same place. I would suggest to follow [sklearn example ](https://github.com/scikit-learn/scikit-learn/tree/main/sklearn)and place in a `exceptions.py` file at the python worker root level.